.env

```diff
@@ -26,6 +26,9 @@ REACT_APP_FIREBASE_VAPID_KEY=BCH2laZcZj5fJyW2Od-iXyAy8OhJ2jpJDWJornW6JDSOi29IFeN
 REACT_APP_NOTIFICATION_API=https://notification.dev.kyberengineering.io/api
 
 REACT_APP_CAMPAIGN_BASE_URL=https://campaigns.dev.kyberengineering.io
+REACT_APP_TYPE_AND_SWAP_URL=https://type-swap.dev.kyberengineering.io/api/v1
```

Comment:
dont include /v1 in base url. in future, suppose that BE has /v2, and in FE code, we need code both v1 and v2, then we need an other env?